<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeStyleAddBuiltinIgnoreDialog</class>
 <widget class="QDialog" name="CodeStyleAddBuiltinIgnoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Built-in Assignment</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter the data for a built-in assignment to be ignored:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Left Side:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="E5ClearableLineEdit" name="leftEdit">
     <property name="placeholderText">
      <string>Enter left hand side of assignment</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Right Side:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="E5ClearableLineEdit" name="rightEdit">
     <property name="placeholderText">
      <string>Enter right hand side of assignment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5ClearableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>E5Gui/E5LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CodeStyleAddBuiltinIgnoreDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CodeStyleAddBuiltinIgnoreDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
